skyportal/models.py

```diff
@@ -652,7 +654,7 @@ class Obj(Base, ha.Point):
     classifications = relationship(
         'Classification',
         back_populates='obj',
-        cascade='save-update, merge, refresh-expire, expunge',
+        cascade='save-update, merge, refresh-expire, expunge, delete-orphan',
```

Comment:
it would be really good to have these (`save-update`, `merge`, etc.) outlined in the docs, and why they are necessary in various forms (not necessary for this PR).